apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/RdaPipelineTestUtils.java

```diff
@@ -91,10 +93,17 @@ public static void runTestWithTemporaryDb(Class<?> testClass, Clock clock, Datab
     // the HSQLDB database will be destroyed when this connection is closed
```

Comment:
ditto on HSQL?

Reply:
Ditto on these remaining references will be removed in a future ticket.